modules/helpers.py

```diff
@@ -69,6 +69,13 @@ def target_creator(command_line_object):
 
         if root.tag.lower() == "nmaprun" and root.attrib.get('scanner') == 'nmap':
             print "Detected nmap xml file\n"
+
+            # command line provided ports
+            # in nmap logic, https ports must also be http 
+            http_ports += command_line_object.add_http_ports
+            http_ports += command_line_object.add_https_ports
```

Comment:
Yeah, you're right.  I'll probably look at that to get it changed around, but for the time being this is the best way to handle it in the current state.  Good catch.